R/repo_lists.R

```diff
@@ -0,0 +1,23 @@
+prod_workflows <- c(
+  "RMI-PACTA/pactaverse",
+  "RMI-PACTA/workflow.data.preparation",
+  "RMI-PACTA/workflow.portfolio.parsing",
+  "RMI-PACTA/workflow.prepare.pacta.indices",
+  "RMI-PACTA/workflow.scenario.preparation",
+  "RMI-PACTA/workflow.transition.monitor"
+)
+
+r2dii.repos <- c(
+  "RMI-PACTA/r2dii.data",
+  "RMI-PACTA/r2dii.match",
+  "RMI-PACTA/r2dii.analysis",
+  "RMI-PACTA/r2dii.plot"
+)
+
+experimental_workflows <- c(
+  "RMI-PACTA/workflow.mfm2023",
+  "RMI-PACTA/workflow.pacta",
+  "RMI-PACTA/workflow.pacta.data.qa",
+  "RMI-PACTA/workflow.pacta.report"
+)
+
```

Comment:
This file is missing all of the PROD transition monitor R packages: 
e.g.
``` r
transition_monitor_repos <- c(
  "RMI-PACTA/pacta.data.validation",
  "RMI-PACTA/pacta.scenario.data.preparation",
  "RMI-PACTA/pacta.data.scraping",
  "RMI-PACTA/pacta.data.preparation",
  "RMI-PACTA/pacta.portfolio.import",
  "RMI-PACTA/pacta.portfolio.audit",
  "RMI-PACTA/pacta.portfolio.allocate",
  "RMI-PACTA/pacta.executive.summary",
  "RMI-PACTA/pacta.portfolio.utils"
)
```

Reply:
No, those are all dependencies of the workflows, so they'll show up as `prod` in the dependency tree

Reply:
More clearly, I explicitly did not include them here, since their "prod-critical" status is entirely defined by their role as a dependency

Reply:
Got it